### PR TITLE
PA-13935 Updated Docker & Amazon deploy actions

### PR DESCRIPTION
**Pull request by SOOS-MMalony**
### Changes:

Updated Docker & Amazon deploy actions

**Ticket:** https://soos.atlassian.net/browse/PA-13935

### CODE REVIEW

**Comment by SOOS-MMalony** [comment]
Should I bump package.json version even though it's just the github action updates?

**Review by SOOS-GSteen — APPROVED** [review]

File: .github/workflows/soos-dast-docker-image-beta.yml [hunk 1]
```diff
@@ -22,10 +22,9 @@ jobs:
...
-        with:
-          context: .
-          push: ${{ github.event_name != 'pull_request' }}
-          tags: soosio/dast:beta
-          labels: ${{ steps.meta.outputs.labels }}
+      - name: Promote alpha to beta
+        id: promote-alpha-to-beta
+        run: |
+          docker pull soosio/dast:alpha
+          docker tag soosio/dast:alpha soosio/dast:beta
```
Comment (SOOS-GSteen) [inline .github/workflows/soos-dast-docker-image-beta.yml:29]:
```suggestion
          docker tag soosio/dast:alpha beta
```
I think the second part doesn't have the image name... at least that is what I did locally

File: .github/workflows/soos-dast-docker-image-latest.yml [hunk 2]
```diff
@@ -21,13 +21,12 @@ jobs:
...
-        with:
-          context: .
-          push: ${{ github.event_name != 'pull_request' }}
-          tags: soosio/dast:latest
-          labels: ${{ steps.meta.outputs.labels }}
+      - name: Promote beta to latest
+        id: promote-beta-to-latest
+        run: |
+          docker pull soosio/dast:beta
+          docker tag soosio/dast:beta soosio/dast:latest
```
Comment (SOOS-GSteen) [inline .github/workflows/soos-dast-docker-image-latest.yml:28]:
```suggestion
          docker tag soosio/dast:beta latest
```

File: .github/workflows/soos-dast-docker-image-beta.yml (hunk 1 above)
Comment (SOOS-MMalony) [reply .github/workflows/soos-dast-docker-image-beta.yml:29]: This is what I get for `docker tag soosio/dast:alpha beta`



Vs `docker tag soosio/dast:alpha soosio/dast:beta`

Comment (SOOS-GSteen) [reply .github/workflows/soos-dast-docker-image-beta.yml:29]: 👍 maybe it was different in the docker hub context

**Pull request merged**